sdk/keyvault/Azure.ResourceManager.KeyVault/src/Generated/Models/VaultListResult.cs

```diff
@@ -12,18 +12,18 @@
 namespace Azure.ResourceManager.KeyVault.Models
 {
     /// <summary> List of vaults. </summary>
-    internal partial class KeyVaultListResult
+    internal partial class VaultListResult
```

Comment:
nit: Since these are internal this isn't a breaking change but was this rename required or a typo from refactoring the autorest.md?

Reply:
Well I would like to reduce the lines of the config therefore I left this off since it is internal.